person.rb

```diff
@@ -22,6 +23,11 @@ def can_use_services?
     @age || @parent_permission
   end
 
+  def add_rentals(rental)
```

Comment:
- [ ] Again, `rental` (in your case _rents_) should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ it. Kindly ensure to improve upon this aspect.